tck/src/main/java/org/eclipse/microprofile/health/tck/NoProcedureSuccessfulTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2017 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICES file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+package org.eclipse.microprofile.health.tck;
+
+import org.eclipse.microprofile.health.tck.deployment.SuccessfulEmptyCheck;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import static org.eclipse.microprofile.health.tck.TCKConfiguration.getHealthURL;
+
+/**
+ * Validates that a health check that includes no procedures returns a 200 status with no body.
+ * @author Scott Stark
+ */
+public class NoProcedureSuccessfulTest extends SimpleHttp {
+
+    @Deployment
+    public static Archive getDeployment() throws Exception {
+        WebArchive deployment = ShrinkWrap.create(WebArchive.class, "tck.war");
+        deployment.addClass(SuccessfulEmptyCheck.class);
```

Comment:
This test, just without the `SuccessfulEmptyCheck.class` added to the war is what I would consider a  deployment "without health check procedures" installed. So I'd recommend to change it that way and rollback and related changes `HealthCheckResponse` which I think are unnecessary.  

All the other assertions however in this test, seem to be correct.